handbook: Adjust wording of statement in updating/upgrading (#441)
freebsd/freebsd-doc: documentation/content/en/books/handbook/cutting-edge/_index.adoc

== Building on non-FreeBSD Hosts

> Historically, building FreeBSD required a FreeBSD host.

Reviewer: ```suggestion
Historically, building required a FreeBSD host.
```

> Nowadays, FreeBSD can be built on Linux distributions and macOS.

Reviewer: ```suggestion
Nowadays, FreeBSD can be built on Linux and macOS.
```

Technical neat to say _Linux distributions_, however yer average reader will understand what's meant by _Linux_.

> To build FreeBSD on non-FreeBSD hosts, the recommendation is to use the `tools/build/make.py` script.

Reviewer: ```suggestion
To build on a non-FreeBSD host, the `tools/build/make.py` script is recommended.
```
